Mobile-Buy-SDK.podspec

```
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name                = 'Mobile-Buy-SDK'
-  s.version             = '3.7.1'
+  s.version             = '3.8.0'
```

The change in `TokenizedPaymentInputV2` (keyword parameter re-order) is considered a breaking change in iOS world. I would suggest keeping the "breaking" change and bumping the major version here. 

```suggestion
  s.version             = '4.0.0'
```

---

What do you think of matching the version in Android too? So we always have the same version? It was something that Ivan had suggested. Android is going to 5.0.0

---

I wanted feedback on the version number before shipping this, thanks!